client/blocks/reader-feed-item/index.tsx

```diff
@@ -44,13 +48,20 @@ export default function ReaderFeedItem( props: ReaderFeedItemProps ): JSX.Elemen
 	} = props;
 	const isWpcomFeed = !! blogId;
 	const translate = useTranslate();
-	const isEmailVerified = useSelector( isCurrentUserEmailVerified );
 	const dispatch = useDispatch();
-	const {
-		isPending: isSubscribing,
-		isSuccess: isSubscribed,
-		mutate: onSubscribe,
-	} = SubscriptionManager.useSiteSubscribeMutation();
+	const isEmailVerified = useSelector( isCurrentUserEmailVerified );
+	const { isPending: isSubscribing, mutate: onSubscribe } =
+		SubscriptionManager.useSiteSubscribeMutation();
+	const { isPending: isUnsubscribing, mutate: onUnsubscribe } =
+		SubscriptionManager.useSiteUnsubscribeMutation();
+
+	// Get feed subscription data. This is used to determine if the user is already subscribed to the feed.
+	useEffect( (): void => {
+		dispatch( requestFollows() ); // Update feed subscriptions.
+	} );
```

Comment:
Do we just need this to run on mount? Currently with no dep array it will request every render cycle.

Reply:
Thinking about this more, does every individual reader-feed-item need to request all follows (regardless of mount or render) ?

Should we be dispatching this from a parent higher up?

Reply:
I have also noted it as unnecessary but the fix was in the other PR. Now I have ported it into the current PR [55f876c](https://github.com/Automattic/wp-calypso/pull/101042/commits/55f876c6b9ab9fa063ca2ba87917e89bee6ed908). Please have a look 🙂